.github/ISSUE_TEMPLATE/rubber-duck.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Rubber Duck Template 🦆
+about: Use for students needing to think things out
+title: ''
+labels: help wanted
```

Comment:
We also want to add a "Rubber Duck" label so that we can know what the ticket type is.

Reply:
Good catch